<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskPanel</class>
 <widget class="QWidget" name="TaskPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>384</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>384</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>128</height>
        </size>
       </property>
       <property name="title">
        <string>Add excitation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="sortLabel">
          <property name="text">
           <string>Select the type of excitation you would like to add.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="sortList"/>
        </item>
        <item>
         <widget class="QLabel" name="systemLabel">
          <property name="text">
           <string>Select which system this subsystem should be added to.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="systemList"/>
        </item>
        <item>
         <widget class="QLabel" name="subsystemFromLabel">
          <property name="text">
           <string>Select which subsystem this excitation should be added to.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="subsystemFromList"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
